ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_path_groups.schema.yml

```diff
@@ -49,3 +49,16 @@ keys:
               local:
                 type: str
                 description: Optional, if not set, the path-group `name` is used as local.
+        default_preference:
+          type: str
+          convert_types:
+            - int
+          description: |-
+            Preference to be used in auto-generated load balance policies.
```

Comment:
This would also be the default preference if a preference is not given in a policy.